src/data/faq_de.json

```
@@ -1581,17 +1601,19 @@
                                     "<ul><li>das entsprechende Labor noch nicht vollständig mit dem Backend der Corona-Warn-App zusammenarbeitet.</li><li>die Übermittlungsfreigabe des Testergebnisses an die Corona-Warn-App über das entsprechende Feld im Laborauftragsformular <a href='/assets/documents/MU_10C_(01_2021)_web.pdf' target='_blank' >10C</a>/<a href='/assets/documents/MU_OEGD_(08_2021)_web.pdf' target='_blank' >OEGD</a> zum Testen nicht erteilt wurde.</li></ul>",
                                     "Erkundigen Sie sich daher bitte, ob das Labor, mit dem der Arzt oder das Testcenter zusammenarbeiten, an die Corona-Warn-App angebunden ist und das Feld zur Übermittlungsfreigabe auf dem Formular angekreuzt ist. Nur dann ist sichergestellt, dass das genutzte Verfahren mit der Corona-Warn-App funktioniert.",
                                     "Derzeit sind 90% der vorhandenen Testkapazitäten in Deutschland durch den kompletten digitalen Prozess abgedeckt. Wir arbeiten mit den Laboren weiter daran, die Lücken zu schließen und alle Labore für den Prozess anzubinden und aufzurüsten. Deshalb kann es vorkommen, dass ein Labor zwar die Beauftragung für den Test mit dem richtigen Formular bekommt, das Testergebnis aber auf einem anderen Weg an die getestete Person übermitteln muss.",
-                                    "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die TAN-Hotline eine teleTAN beantragen (<a href='#international_phone_numbers' target='_blank' >klicken Sie hier um zu den Telefonnummern zu gelangen</a>), um über die Corona-Warn-App Ihre Kontakte zu warnen. Bevor Sie die teleTAN in die App eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
+                                    "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die TAN-Hotline eine teleTAN beantragen (<a href='#international_phone_numbers' target='_blank' >klicken Sie hier um zu den Telefonnummern zu gelangen</a>), um über die Corona-Warn-App Ihre Kontakte zu warnen. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt. Bevor Sie die teleTAN in die App (Version 2.28 oder kleiner) eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",

```

Specified placeholder was ??????h, one ? is missing

Version 2.28 oder kleiner -> Version 2.28 oder darunter 
```suggestion
                                    "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die TAN-Hotline eine teleTAN beantragen (<a href='#international_phone_numbers' target='_blank' >klicken Sie hier um zu den Telefonnummern zu gelangen</a>), um über die Corona-Warn-App Ihre Kontakte zu warnen. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt. Bevor Sie die teleTAN in die App (Version 2.28 oder darunter) eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
```